packages/collector/test/tracing/database/ioredis/app.js

```diff
@@ -15,7 +15,18 @@ process.on('SIGTERM', () => {
 
 const agentPort = process.env.INSTANA_AGENT_PORT;
 
-require('../../../..')();
+const ignoreEndpointsEnabled = process.env.IGNORE_ENDPOINTS === 'true';
+if (!ignoreEndpointsEnabled) {
+  require('../../../..')();
+} else {
+  require('../../../..')({
+    tracing: {
+      ignoreEndpoints: {
```

Comment:
issue: Please add support for env variable `INSTANA_IGNORE_ENDPOINTS`. It's absolutely fine if Java has no support for it yet. Then we don't have to do this special logic here in all apps.

Reply:
I’ve added the environment variable, `INSTANA_IGNORE_ENDPOINTS`, and stored it in JSON string format for now, keeping it internal.